lib/mix/test/fixtures/deps_status/deps/ok/mix.exs

```diff
@@ -5,4 +5,8 @@ defmodule Ok.Mixfile do
     [app: :ok,
      version: "0.1.0"]
   end
+
+  def application do
+    [extra_applications: [:logger]]
```

Comment:
Shouldn't we now remove the `:logger` from somewhere to effectively test this change?